src/handler.ts

```diff
@@ -6,39 +6,48 @@ import { sendMCProhibition } from './eventbridge/Send';
 import logger from './observability/Logger';
 import { MCRequest } from './utils/MCRequest';
 
-const { NODE_ENV, SERVICE, AWS_REGION, AWS_STAGE } = process.env;
+const { NODE_ENV, SERVICE, AWS_REGION, AWS_STAGE, SEND_TO_SMC } = process.env;
 
 logger.debug(
-  `\nRunning Service:\n '${SERVICE}'\n mode: ${NODE_ENV}\n stage: '${AWS_STAGE}'\n region: '${AWS_REGION}'\n\n`,
+  `\nRunning Service:\n '${SERVICE}'\n mode: ${NODE_ENV}\n stage: '${AWS_STAGE}'\n region: '${AWS_REGION}'\n 
+  Send to smc: '${SEND_TO_SMC}\n`,
 );
 
 const handler = async (
   event: DynamoDBStreamEvent,
   _context: Context,
   callback: Callback,
 ) => {
-  try {
-    logger.debug(`Function triggered with '${JSON.stringify(event)}'.`);
+  if (
+    process.env.SEND_TO_SMC != undefined &&
+    process.env.SEND_TO_SMC.toUpperCase() == 'TRUE'
+  ) {
+    try {
+      logger.debug(`Function triggered with '${JSON.stringify(event)}'.`);
 
-    // We want to process these in sequence to maintain order of database changes
-    for (const record of event.Records) {
-      const mcRequests: MCRequest[] = extractMCTestResults(record);
-      if (mcRequests != null) {
-        await sendMCProhibition(mcRequests);
+      // We want to process these in sequence to maintain order of database changes
+      for (const record of event.Records) {
+        const mcRequests: MCRequest[] = extractMCTestResults(record);
+        if (mcRequests != null) {
+          await sendMCProhibition(mcRequests);
+        }
+      }
+      callback(null, 'Data processed successfully.');
+    } catch (error) {
+      if (error.body) {
+        logger.error(JSON.stringify(error.body));
+        callback(
+          null,
+          `Data processed unsuccessfully: ${JSON.stringify(error.body)}`,
+        );
+      } else {
+        logger.error(error);
+        callback(null, `Data processed unsuccessfully: ${error}`);
       }
     }
-    callback(null, 'Data processed successfully.');
-  } catch (error) {
-    if (error.body) {
-      logger.error(JSON.stringify(error.body));
-      callback(
-        null,
-        `Data processed unsuccessfully: ${JSON.stringify(error.body)}`,
-      );
-    } else {
-      logger.error(error);
-      callback(null, `Data processed unsuccessfully: ${error}`);
-    }
+  } else {
+    logger.info('Missing or not true environment variable present');
```

Comment:
I wonder if it is worth slightly amending the log so we can quickly see what that not true/missing environment variable results in, ie...

"SMC Prohibition Function deactivated due to falsy/missing environment variable" or something similar maybe?

Reply:
I don't think we need to include the log as it will only appear in the SMC prohibition log when triggered via AWS. However, i'm happy to amend it to say what it results in